contracts/LargeStorageManager.sol

```diff
@@ -50,7 +51,11 @@ contract LargeStorageManager {
 
         if (keyToMetadata[key][chunkId] == bytes32(0)) {
             keyToChunkNum[key]++;
+        } else {
```

Comment:
Not a consistent code since it is handled in the _prepare method in BlobStorageManager.sol

Reply:
fix